internal/testrunner/runners/system/servicedeployer/compose.go

```diff
@@ -158,6 +158,10 @@ func (s *dockerComposeDeployedService) TearDown() error {
 		if err != nil {
 			logger.Errorf("could not remove the service logs (path: %s)", s.ctxt.Logs.Folder.Local)
 		}
+		// Remove the outputs generated by the service container
+		if err = os.RemoveAll(s.ctxt.OutputDir); err != nil {
+			logger.Errorf("could not remove the temporary output files %w", err)
+		}
```

Comment:
Should this be run just when `s.ctxt.OutputDir` is different from empty string ?

Reply:
If it is an empty string it does nothing 
https://github.com/golang/go/blob/094c75219a160be1667c9757f363ffad8926632b/src/os/removeall_at.go#L15-L20